comms/internal/rpcz/apply.go

```diff
@@ -12,8 +12,11 @@ import (
 	"github.com/nats-io/nats.go"
 )
 
-// Validates + Applies a NATS message
+var (
+	JetstreamClient nats.JetStreamContext
```

Comment:
Had to create a duplicate `JetstreamClient` in `rpcz` because the original `JetstreamClient` lives in `peering` and `rpcz` is a dependency of `peering`; get a circular dependency error when importing `peering` in `rpcz`